### DISCUSSION TITLE
EXTREME osu!lazer lag issues

### DISCUSSION THREAD

**Discussion by zMienai** [ppy/osu#22810] [Q&A] [answered]
hi! i had issues with lag on osu!lazer but it was still playable just fine, but today something happened to my osu!lazer. Everything (except for my exports and screenshots) got wiped, so i copied my backup osu!lazer files, but now it lags so bad, it's unplayable. Does anyone know how to reduce lag so it's playable again?

NOTE: idk if this has anything to do with the wipe and lag happening, but i remember that i was trying to free up some space on my computer by deleting some stuff in places like appdata, temp, etc...

**Comment by Ladwick** [comment]
Have you tried to uninstall and reinstall the app? Maybe after you did the cleaning, you accidentally eliminated something that lazer uses, so maybe you should start by reinstalling the app?

**Reply by zMienai** [reply]
> Have you tried to uninstall and reinstall the app? Maybe after you did the cleaning, you accidentally eliminated something that lazer uses, so maybe you should start by reinstalling the app?

I did. 2x. Didn't work. when i did it the second time, it got a tad bit better. still not as good as before tho, so i would not say that this is resolved. But i know why i lag so bad. It's cause i have to play on Vsync cause it makes me lag the least. Before i played on basically unlimited. So if that is fixed, then i should be fine again

**Comment by bdach** [comment]
Can't even begin to diagnose this without any data like logs, video of this "extreme lag" (preferably with frame graph showing; press ctrl-f11 twice)

**Reply by zMienai** [reply]
Sorry, but i am not able to record it. An easier way to put it is that the "Basically unlimited" frame mode is laggier than "Vsync" and i want to fix it

**Comment by zMienai** [comment, marked as answer]
Ok so turns out, it fixed itself. Time was all it needed.
